internal/pkg/otel/elasticdiagnosticsextension/config.go

```diff
@@ -0,0 +1,25 @@
+package elasticdiagnosticsextension
```

Comment:
The extension should have a readme document its design, ideally with a simple diagram showing the socket communication paths